uobtheatre/addresses/models.py

```diff
@@ -13,6 +13,12 @@ class Address(models.Model):
     street = models.CharField(max_length=255)
     city = models.CharField(max_length=255)
     postcode = models.CharField(max_length=9)
+    what3words = models.CharField(
+        max_length=255,
```

Comment:
For info, I believe the (max W3W address length is 64 chars)[https://developer.what3words.com/design/maker-guidelines#the-basics]